html/doc/faq.html

```diff
@@ -698,7 +698,36 @@ <h2 id="collapse-newlines">Why won't the <code>collapse_whitespace</code> filter
 are <code>display:&nbsp;inline</code>.
 </p>
 
+<h2 id="warning-fetch-rate">I've got a warning saying
+"Serf fetch failure rate extremely high". What does this mean?</h2>
 
+<p>The warning means that when PageSpeed tried to fetch resources inside your
+web page for optimization over 50% of attempts inside a 30-minute period
+failed. This may just mean you have some broken resource includes in your
+pages (in which case, it may be a good idea to fix them for better performance),
+but might indicate that PageSpeed's fetching is not working right. If you have
+in-place resource optimization on, that can result in things failing
+intermittently.</p>
```

Comment:
reference bug here, and/or be more specific, such as 404 responses from .pagespeed. resources?